pkg/broker/broker.go

```diff
@@ -953,6 +953,16 @@ func (a AnsibleBroker) Unbind(
 		return nil, false, errors.New(errMsg)
 	}
 
+	jobInProgress, jobToken, err := a.isJobInProgress(&instance, apb.JobMethodUnbind)
+	if err != nil {
+		log.Errorf("An error occurred while trying to determine if a unbind job is already in progress for instance: %s", instance.ID)
+		return nil, false, err
+	}
+	if jobInProgress {
+		log.Infof("Unbind requested for instance %s, but job is already in progress", instance.ID)
+		return &UnbindResponse{Operation: jobToken}, false, ErrorDeprovisionInProgress
```

Comment:
Good point. Ive added logic to the handler to check the error but I think returing 202 regardless of async or not is appropriate since the job in running and it will force the client to poll last op endpoint for the result. This meets the spec `202 MUST be returned if the unbinding is in progress.` which in this case it is. 

This logic is also used for the deprovision handler